templates/store.html

```diff
@@ -11,23 +11,73 @@ <h1 class="p-heading--2">The app store for clouds</h1>
     <p>Find and deploy <a href="" class="p-link--inverted">Juju</a> solutions.</p>
   </div>
 </section>
+<div class="p-sort" data-js="mobile-sort-handler">
+  <div class="p-sort__title">
+    Sort by
+  </div>
+  <form class="p-sort__content">
+    <div class="p-sort__item">
+      <input type="radio" name="sortMobile" id="featured" value="featured" checked="">
```

Comment:
this should use the context provided by the python to have the correct item checked on page load